accounts-db/src/ancient_append_vecs.rs

```diff
@@ -591,6 +587,14 @@ impl AccountsDb {
             })
             .count()
             .saturating_sub(randoms as usize);
+        // ideal storage size is total alive bytes of ancient storages / half of max ancient slots
+        tuning.ideal_storage_size = NonZeroU64::new(
+            (infos.total_alive_bytes.0 / tuning.max_ancient_slots.max(1) as u64 * 2).max(5_000_000),
```

Comment:
this works out to 60MB per file if we assume 300GB of alive bytes and 10k slots. If we have 100k slots, then this results in 6MB per file. The hardcoded previous size was 130MB I believe. These numbers are all in a reasonable range.